pool/app/pool_common/locales/i18n_en.ml

```diff
@@ -203,6 +203,7 @@ let nav_link_to_string = function
   | PrivacyPolicy -> "Privacy policy"
   | Profile -> "Profile"
   | Queue -> "Queued jobs"
+  | QueueHistory -> "Executed jobs"
```

Comment:
Thought: A cancelled job wasn't necessarily executed; rename it to "Job history".

```suggestion
  | QueueHistory -> "Job history"
```